src/timeline/Timeline.js

```diff
@@ -30,7 +30,10 @@ export default class Timeline extends React.PureComponent {
         summary: PropTypes.string.isRequired,
         color: PropTypes.string
       })
-    ).isRequired
+    ).isRequired,
+    scrollToNow: PropTypes.bool,
+    currentDateString: PropTypes.string,
+    updateCurrentTimeIndicatorEveryMinute: PropTypes.bool,
```

Comment:
I think it's better to let the user decide how often to update by passing the number of seconds/minutes, like 'updateInterval' or something like that (where default is 0 - no updates).